src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -149,6 +149,35 @@ def acr_build_task_list(cmd,
         cmd.cli_ctx, registry_name, resource_group_name, BUILD_TASKS_NOT_SUPPORTED)
     return client.list(resource_group_name, registry_name)
 
+def acr_build_arguments_list(cmd,
+                             client,
+                             registry_name,
+                             build_task_name,
+                             resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, BUILD_TASKS_NOT_SUPPORTED)
+    return client.list_build_arguments(resource_group_name=resource_group_name,
+                                       registry_name=registry_name,
+                                       build_task_name=build_task_name,
+                                       step_name=_get_build_step_name(build_task_name))
+
+def acr_build_source_repository_properties_list(cmd,
+                                                client,
+                                                registry_name,
+                                                build_task_name,
+                                                resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, BUILD_TASKS_NOT_SUPPORTED)
+    return client.list_source_repository_properties(resource_group_name, registry_name, build_task_name)
+
+def acr_build_task_cancel(cmd,
+                          client,
+                          registry_name,
+                          build_id,
```

Comment:
Do we want to support cancel last created build (in a build task) style? similar to logs.
/cc @ankurkhemani @northtyphoon @ehotinger

Reply:
well cancellation is still not implemented AFAIK, unless someone is actively implementing it. So no cancellation should be in the CLI yet

Reply:
Let's remove it from this PR then and ship the other three commands.